### DISCUSSION TITLE
What A Coincidence

### DISCUSSION THREAD

**Discussion by danii** [DzenanJupic/rustube#2] [General]
What a happy little coincidence this is.
![image](https://user-images.githubusercontent.com/39541871/106655069-e0f2a500-6566-11eb-908f-74501d58a635.png)
I barely look at [crates.io](https://crates.io/), I primarily use [lib.rs](https://lib.rs/), and the one time I checked that so happened to be there, lol.
Small world I guess.

**Comment by DzenanJupic** [comment]
Nice 😁 that is, indeed, quite a coincidence.

What do you think about the CLI? I'm actually not totally sure which features to include and if the current interface is too clumsy.
